fullstop-job-launcher/pom.xml

```diff
@@ -1,24 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
this file contains only changes in linebreaks. Was this intended?

Reply:
reverted